app/src/pages/organization/projectTeamPage/membersPageHeader/membersPageHeader.jsx

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import Parser from 'html-react-parser';
+import classNames from 'classnames/bind';
+import searchIcon from 'common/img/newIcons/search-outline-inline.svg';
+import filterIcon from 'common/img/newIcons/filters-outline-inline.svg';
+import { Button } from 'componentLibrary/button';
+import { useIntl } from 'react-intl';
+import { messages } from '../messages';
+import styles from './membersPageHeader.scss';
+
+const cx = classNames.bind(styles);
+
+export const MembersPageHeader = ({ hasPermission, isNotEmpty }) => {
```

Comment:
The same comment about naming consistency.

Reply:
Done.